test/WormholeL2GovernorMetadata.t.sol

```diff
@@ -21,7 +22,8 @@ contract L2GovernorMetadataTest is Constants {
 
 contract Constructor is L2GovernorMetadataTest {
   function testFuzz_CorrectlySetsAllArgs(address wormholeCore) public {
-    new WormholeL2GovernorMetadata(wormholeCore); // nothing to assert as there are no constructor
+    new WormholeL2GovernorMetadata(wormholeCore, msg.sender); // nothing to assert as there are no
+      // constructor
```

Comment:
nit: "args set" on next line should prob collapse onto this line

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/55/commits/125984e93a58f95a7e2726afcbee6335b44bac29